app/views/steps/case/date_stamp/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<% title t('.page_title') %>
+<% step_header %>
+
+<div class="govuk-grid-row app-inverted--card app-inverted-text">
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-l app-inverted-text">Your application has been date stamped</h1>
+
+    <div class="govuk-button-group">
+      <%= link_to '#',
```

Comment:
You can use `#link_button` helper, it adds some additional attributes for you. You can still provide any additional classes like `app-button--inverted`.

For the locales maybe we can use the existing `helpers.submit.save_and_continue` and `helpers.submit.save_and_come_back_later` to keep it in one place.